### ISSUE TITLE
Staging Review finding: Input error messages not announced by screen readers

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#89010]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Modernized Check-in
**Product:** Pre-Check-in & eCheck-in
**Feature:** New Landing page design

## Findings details
**VA.gov Experience Standard - issue:** [User doesn't have enough information to complete a task.](https://design.va.gov/about/experience-standards/#user-has-enough-information-to-complete-a-task)
**VA.gov Experience Standard - category:** [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)
**Launch-blocking:** Yes
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

On /appointment-check-in/verify, if a user skips one of the required inputs and then clicks Continue, an error message appears in line with the input that was skipped.

However, that message isn't announced by screen readers. If a screen reader user accidentally skips the input, they won't receive any feedback to tell them what's wrong. They may explore the page to try to figure out why they're not able to continue, or they may assume the check-in tool is broken and give up.

Note that I didn't think to check this on the pre-check-in page, but I would recommend double-checking there for similar behavior.

<details><summary>Link, screenshot or steps to recreate</summary>

https://github.com/user-attachments/assets/a71913e7-988c-44b8-94fc-1c215937212a

</details>

## Recommended action

On most VA forms when there's a validation error when clicking Continue, keyboard focus is sent to the first input showing an error. That ensures that the input label and error are immediately announced by screen readers, and puts users in the right place to fix it. I would recommend matching behavior, like [40-0247](https://staging.va.gov/burials-memorials/memorial-items/presidential-memorial-certificates/request-certificate-form-40-0247/) as an easy example.

## References

- **Accessibility Defect Severity:** [1: Critical. Must be fixed before launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-1)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by loripusey** [comment]
Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Check-in-Experience-61fc23a2cb8a14001132e102/issues/department-of-veterans-affairs/va.gov-team/89010?planning-poker) with Zenhub @brianseek

**Comment by loripusey** [comment]
Can reach out to Nick Sprinkle for additional info?

**Comment by loripusey** [comment]
Fixed & merged
